mlflow/tracing/processor/mlflow.py

```diff
@@ -107,37 +107,18 @@ def _start_trace(self, span: OTelSpan) -> TraceInfo:
             tags.update({TraceTagKey.EVAL_REQUEST_ID: request_id})
         if depedencies_schema := maybe_get_dependencies_schemas():
             tags.update(depedencies_schema)
-        try:
-            trace_info = self._client._start_tracked_trace(
-                experiment_id=experiment_id,
-                # TODO: This timestamp is not accurate because it is not adjusted to exclude the
-                #   latency of the backend API call. We do this adjustment for span start time
-                #   above, but can't do it for trace start time until the backend API supports
-                #   updating the trace start time.
-                timestamp_ms=span.start_time // 1_000_000,  # nanosecond to millisecond
-                request_metadata=metadata,
-                tags=tags,
-            )
 
-        # TODO: This catches all exceptions from the tracking server so the in-memory tracing
-        # still works if the backend APIs are not ready. Once backend is ready, we should
-        # catch more specific exceptions and handle them accordingly.
-        except Exception:
```

Comment:
note: Now the backend is fully up, so we should not have this fallback client-side generation. The backend doesn't accept client-side generated request ID and this fallback trace is not logged after all, so it just complicates debugging.

Reply:
Nice!